pkg/epp/backend/metrics/logger.go

```diff
@@ -86,19 +86,19 @@ func StartMetricsLogger(ctx context.Context, datastore Datastore, refreshPrometh
 	}
 }
 
-func flushPrometheusMetricsOnce(logger logr.Logger, datastore Datastore) {
+func refreshPrometheusMetrics(logger logr.Logger, datastore Datastore) {
 	pool, err := datastore.PoolGet()
 	if err != nil {
 		// No inference pool or not initialize.
-		logger.V(logutil.DEFAULT).Info("pool is not initialized, skipping flushing metrics")
+		logger.V(logutil.DEBUG).Info("Pool is not initialized, skipping refreshing metrics")
```

Comment:
This should be transitory, I think it is useful to report this in default mode.

Reply:
reverted